nservicebus/azure-service-bus/keep-connectionstring-safe.md

```diff
@@ -0,0 +1,32 @@
+---
+title: Keep connection string safe
+summary: Configuring Azure ServiceBus transport to secure connection strings. 
+component: ASB
+tags:
+- Cloud
+- Azure
+- Transports 
+- Security
+reviewed: 2016-04-26
+---
+
+include: keep-connection-string-safe
+
+To enable this behavior, apply the following configuration:
```

Comment:
Overlooked the `keep-connection-string-safe` part.
Would it make sense to elaborate more and provide a little example (tiny snippet of an output) to demonstrate the problem?